src/main/java/games/strategy/engine/lobby/server/db/BannedMacController.java

```diff
@@ -34,30 +27,21 @@ public void addBannedMac(final String mac, final Instant banTill) {
     if (isMacBanned(mac).getFirst()) {
       removeBannedMac(mac);
     }
```

Comment:
@ssoloff Here for example: If the MAC is already present it will be removed instead.
Possible solutions would be to either pass an Instant in the past to remove the mac, or just use Instant.MAX_VALUE instead of null for forever

Reply:
However this would introduce breaking changes as the parameters are sent over RMI

Reply:
Nevermind, there's no return statement which I originally assumed.
Writing tests reveals what methods are actually doing 😅